### PR TITLE
Android: External Storage write permissions

### PR DESCRIPTION
**Pull request by redwolf2**

### CODE REVIEW

File: unity/ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -137,7 +137,7 @@ PlayerSettings:
   protectGraphicsMemory: 0
   useHDRDisplay: 0
   applicationIdentifier:
-    Android: com.Bruce.Valkyrie
+    Android: com.bruce.valkyrie
```
Comment (redwolf2) [inline unity/ProjectSettings/ProjectSettings.asset]: Android package names should be lower case.

**Comment by NPBruce** [comment]
Please target the development branch (only difference right now is the version number), thanks.

**Comment by redwolf2** [comment]
Done

File: unity/Assets/Resources/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-1.5.0a
```
Comment (NPBruce) [inline unity/Assets/Resources/version.txt]: Please change this back (or change to 1.6.0.a).

Comment (redwolf2) [reply unity/Assets/Resources/version.txt]: Changed to 1.6.0a (without the last dot)

**Pull request merged**